Runtime/Plugins/Raycast/RaycastManager.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections.Generic;
+using Unity.UIWidgets.ui;
+using UnityEngine;
+using Rect = Unity.UIWidgets.ui.Rect;
+
+namespace Unity.UIWidgets.plugins.raycast {
+    public class RaycastableRect {
+        public bool isDirty;
```

Comment:
it could be better to seal up these values and update their value by methods only ? for instance,  a getter for isDirty, and three methods, i.e., markDirty(), updateRect() and checkInRect()